### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by megatimranikram**

### CODE REVIEW

**Comment by megatimranikram** [comment]
ready

**Comment by ngocvt-0484** [comment]
ready

**Comment by ngocvt-0484** [comment]
commented

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ruby '2.7.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+# Use mysql2 as the database for Active Record
+gem 'mysql2', '>= 0.4.4'
+# Use Puma as the app server
+gem 'puma', '~> 4.1'
+# Use SCSS for stylesheets
+gem 'sass-rails', '>= 6'
```
Comment (ngocvt-0484) [inline Gemfile]: use " plz

**Comment by megatimranikram** [comment]
Ready

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: using ' and remove all the comment lines

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: using "

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MalayGroup4</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: using "

File: config/database.yml [hunk 5]
```diff
@@ -0,0 +1,54 @@
+# MySQL. Versions 5.5.8 and up are supported.
```
Comment (saobv-2223) [inline config/database.yml]: image

File: test/application_system_test_case.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```
Comment (saobv-2223) [inline test/application_system_test_case.rb]: image

**Comment by megatimranikram** [comment]
ready

**Pull request merged**